package.json

```diff
@@ -66,18 +66,10 @@
     "node": "10"
   },
   "dependencies": {
-    "@signalk/freeboard-sk": "^1.0.0",
-    "@signalk/instrumentpanel": "0.x",
-    "@signalk/maptracker": "^1.0.0",
-    "@signalk/playground": "^1.0.0",
-    "@signalk/sailgauge": "^1.1.0",
+    "@serialport/bindings": "^8.0.6",
```

Comment:
Was this intentional? serialport should be a streams dependency I think.

Reply:
it's used in src/serverroutes.js